game/src/lib.rs

```diff
@@ -49,8 +49,21 @@ struct Setup {
     start_time: Option<Duration>,
     load_kml: Option<String>,
     diff_map: Option<String>,
-    ungap: bool,
-    ltn: bool,
+    // TODO Fold more of Setup into the mutually exclusive Mode. load_kml, maybe_mode, and
```

Comment:
This file's setup process is increasingly insane. I'll try to simplify it soon. This is also one of the last places awaiting cutover from our weird CLI `CmdArgs` flag processing to `structopt`.